obspy/core/tests/test_util_base.py

```diff
@@ -4,13 +4,14 @@
 from future.builtins import *  # NOQA
 
 import os
+import copy
 import shutil
 import unittest
 
 from obspy.core.compatibility import mock
 from obspy.core.util.base import (NamedTemporaryFile, get_dependency_version,
                                   download_to_file, sanitize_filename,
-                                  create_empty_data_chunk)
+                                  create_empty_data_chunk, ComparingObject)
```

Comment:
This sorts at the very beginning of the `import` list.

Reply:
Thanks for the review.
Sorry, I don't understand what this comment means. Are you suggesting a change?

Reply:
I don't think we ever bothered to enforce alphabetical order *inside* import attribute lists, we just enforce alphabetical order in the module names being imported from. So if this is what @QuLogic meant I think it can stay as is.

Reply:
oh, gotcha, thanks!